### ISSUE TITLE
Extension doesn't want to run my test when project folder is in a subfolder

### ISSUE THREAD

**Issue by luca1226** [vitest-dev/vscode#101]
**Describe the bug**
Testing tab doesn't run the test when the frontend/vue project is in a subfolder.

**To Reproduce**
1. Clone [the example project](https://github.com/luca1226/vitest-issue-example):
2. Install package with `npm i`
3. Go to the Testing tab
4. Launch Test from there or from the file with the green arrow

**Expected behavior**
All test should run

**Screenshots**
![image](https://user-images.githubusercontent.com/46788089/203030715-fe140980-3e98-4b70-8116-49c713469a2a.png)

**Environment**

- OS: Windows
- VSCode version: 1.70.1
- Vitest version: 0.24.3
- Vitest plugin version: 0.2.34

**Comment by kaipaysen** [comment]
Had the same issue. I added these settings to vscode:

``` json
    // .vscode/settings.json
    "vitest.enable": true,
    "vitest.showFailMessages": true,
    "vitest.commandLine": "npm --prefix frontend run test --",
    "vitest.include": [
        "frontend/test/**/*.{test,spec}.{js,ts}"
    ]
```

**Comment by artyrcheek** [comment]
@kaipaysen That seems to work for "run mode" but for "watch mode"
I get
```
/Users/arthurcheek/.nvm/versions/node/v16.13.2/bin/node ./node_modules/vitest/vitest.mjs --prefix web/frontend run test -- /Users/arthurcheek/Desktop/supercycle-experiments/supercycle-react-test/web/frontend/tests/LocationIndex.test.js --api.port 51602 --api.host 127.0.0.1
Uncaught CACError CACError: Unknown option `--prefix`
```

**Comment by kaipaysen** [comment]
@artyrcheek I get the the same message when trying to debug a test in VSCode. Anyway running the tests from Test Explorer works fine.

**Comment by zxch3n** [comment]
You can try using VS Code workspace as a workaround for now.

**Comment by BrainCrumbz** [comment]
Hello everyone. Can someone show an example of how to use VS Code workspaces in order to implement the proposed workaround?

We currently open a folder from VS Code (not a workspace) containing both client and server apps. Folder contents are similar to the following: 

```
.vscode\
  extensions.json
  settings.json
client\
  package.json
  ...
server\
  package.json
  ...
```

`settings.json` at the moment holds some configuration for this vitest extension, specifically:

```json
    "vitest.enable": true,
    "vitest.commandLine": "npm run test-app-int-adapters --prefix ./client"
```

`test-app-int-adapters` is an npm script available under `client/package.json` to run some set of tests.

Without any configuration, test explorer does not list any test. After setting that configuration, test cases are listed in test explorer. But when running any of them, they just hang with the `Running tests...` message in status bar.

We searched and read other similar issues, we're just writing in this one in case anyone knows how to workaround this.  
TA

**Comment by BrainCrumbz** [comment]
Here's what *Test Output* pane shows after launching one of the test files:

```
> client@0.0.0 test-app-int-adapters

> vitest --dir ./tests/app/int-adapters --environment happy-dom /some-path/client/tests/app/int-adapters/session/session-started.test.ts 61419 127.0.0.1


 DEV  v0.29.2 d:/some-path/client
 ✓ tests/app/int-adapters/session/session-started.test.ts  (1 test) 6ms



 Test Files  1 passed (1)

      Tests  1 passed (1)

   Start at  17:42:16

   Duration  1.79s (transform 1.05s, setup 0ms, collect 1.27s, tests 6ms)





 PASS  Waiting for file changes...

       press h to show help, press q to quit

So apparently the tests are also run correctly, but vitest runs in watch mode, and for the extension they are still running. So I guess it's another issue, not related to current one.
```

**Comment by BrainCrumbz** [comment]
For completeness, [this comment](https://github.com/vitest-dev/vscode/issues/55#issuecomment-1231161530) from another issue pointed our attention to the need for a trailing `--`, as also mentioned in readme [config](https://github.com/vitest-dev/vscode#config) section.

We modified extension settings like the following:

```json
    "vitest.enable": true,
    "vitest.commandLine": "npm run test-app-int-adapters --prefix ./client --"
```

and now the target set of tests runs correctly in test explorer.

**Comment by sheremet-va** [comment]
This should be fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.
